new-lamassu-admin/src/components/NotificationCenter/NotificationCenter.styles.js

```diff
@@ -0,0 +1,130 @@
+import {
+  spacer,
+  white,
+  zircon,
+  secondaryColor,
+  spring3,
+  comet,
+  offColor
+} from 'src/styling/variables'
+
+const styles = {
+  background: {
+    position: 'absolute',
+    width: '100vw',
+    height: '100vh',
+    left: 0,
+    top: 0,
+    zIndex: -1,
+    backgroundColor: white,
+    boxShadow: '0 0 14px 0 rgba(0, 0, 0, 0.24)'
+  },
+  container: {
+    left: -200,
+    top: -42,
+    backgroundColor: white,
+    height: '110vh'
+  },
+  header: {
+    display: 'flex',
+    justifyContent: 'space-between'
+  },
+  headerText: {
+    marginTop: spacer * 2.5,
+    marginLeft: spacer * 3
+  },
+  actionButtons: {
+    display: 'flex',
+    marginLeft: spacer * 2
+  },
+  notificationIcon: {
+    position: 'absolute',
+    left: spacer * 33,
+    top: spacer * 2 + 4,
+    cursor: 'pointer',
+    background: 'transparent',
+    boxShadow: '0px 0px 0px transparent',
+    border: '0px solid transparent',
+    textShadow: '0px 0px 0px transparent',
+    outline: 'none'
+  },
+  clearAllButton: {
+    marginTop: -spacer * 2,
+    marginLeft: spacer,
+    backgroundColor: zircon
+  },
+  notificationsList: {
+    width: 440,
+    height: '90vh',
+    maxHeight: '100vh',
+    marginTop: 8,
+    marginLeft: 0,
+    marginRight: 10,
+    overflowY: 'auto',
+    overflowX: 'hidden',
+    backgroundColor: white,
+    zIndex: 10,
+    '&::-webkit-scrollbar': {
+      width: 7
+    },
+    '&::-webkit-scrollbar-thumb': {
+      backgroundColor: offColor,
+      borderRadius: 5
+    },
+    '&::-webkit-scrollbar-track-piece:end': {
+      background: 'transparent'
+    }
```

Comment:
did you test this in other browsers? Scrollbar is usually a PITA to customize.

Reply:
it works on chrome but not on firefox. Should I remove this?